concreate/templates/template.jinja

```diff
@@ -47,19 +47,19 @@ EXPOSE {%- for port in helper.ports(ports) %} {{ port }}{% endfor %}
 USER root
 
 {% if packages and packages.install %}
-{% if additional_repos %}
+{% if packages.repositories %}
 # Add custom repo files
 COPY repos/*.repo /etc/yum.repos.d/
```

Comment:
We probably should add selectively files here, just like we do it at removal time, so nothing we do not expect will not be added to container and then NOT removed.

Reply:
good catch, thanks